vendor/github.com/getgauge/common/common.go

```diff
@@ -70,15 +70,15 @@ type Property struct {
 func GetProjectRoot() (string, error) {
 	pwd, err := os.Getwd()
 	if err != nil {
-		return "", fmt.Errorf("Failed to find project root directory. %s\n", err.Error())
+		return "", fmt.Errorf("Failed to find Gauge project root directory. Missing manifest.json file: %s\n", err.Error())
```

Comment:
this needs to be changed in https://github.com/getgauge/common/common.go and then vendored into gauge again.

Else, the next time this is vendored in, the changes will be lost. It's best to avoid changing the `vendor` directory manually.